plugins/orchestrator-backend/src/service/router.ts

```diff
@@ -779,7 +779,7 @@ function setupInternalRoutes(
       }
 
       if (!definition.dataInputSchema) {
-        res.status(200).json(res);
+        res.status(200).json({});
```

Comment:
In both cases, is `200` ok? I mean, it is a successful response?

Reply:
yes it's successful
you can have a workflow without an input schema